### DISCUSSION TITLE
Getting Stars Lists Info with REST API

### DISCUSSION THREAD

**Discussion by trganda** [community/community#54240] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I'm looking for a way (the rest api was best) to get the [star lists](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars) information, the information should tell the requester:

- name of the list
- description of the list
- what repositories was included in this list or the resposne data can be combined with [https://api.github.com/users/USERNAME/starred](https://docs.github.com/en/rest/activity/starring?apiVersion=2022-11-28#list-repositories-starred-by-a-user)

**Comment by jay23606** [comment]
graphql might be better for this

**Comment by guibranco** [comment]
Already requested here #8293 in 2021 😞
